playbooks/roles/edxapp/defaults/main.yml

```diff
@@ -1460,6 +1460,7 @@ generic_env_config:  &edxapp_generic_env
   SESSION_COOKIE_NAME:  "{{ EDXAPP_SESSION_COOKIE_NAME }}"
   COMMENTS_SERVICE_KEY: "{{ EDXAPP_COMMENTS_SERVICE_KEY }}"
   TECH_SUPPORT_EMAIL: "{{ EDXAPP_TECH_SUPPORT_EMAIL }}"
+  TECH_SUPPORT_URL: "{{ EDXAPP_TECH_SUPPORT_URL }}"
```

Comment:
The reason why this line is failing in the tests, is because `EDXAPP_TECH_SUPPORT_URL` needs to have a default value, like the other `EDXAPP_` vars here, for example `TECH_SUPPORT_EMAIL` has a default here: https://github.com/appsembler/configuration/pull/375/files#R409 above in the file.

Despite this, I think we can avoid introducing this value here, please see my comment bellow.